src/bin/directoryd.rs

```diff
@@ -35,9 +35,9 @@ struct Cli {
 }
 
 fn main() -> Result<(), DirectoryServerError> {
-    setup_directory_logger(log::LevelFilter::Info);
-
+    setup_directory_logger(log::LevelFilter::Info, None);
```

Comment:
Pass the `args.data_dir` value here. This will make the log dir always fixed to default. Same for all other apps.